Src/VimCore/CoreInterfaces.fs

```diff
@@ -4215,7 +4230,7 @@ type IVimHost =
     abstract LoadFileIntoExistingWindow: filePath: string -> textView: ITextView -> bool
 
     /// Loads the new file into a new existing window
-    abstract LoadFileIntoNewWindow: filePath: string -> bool
+    abstract LoadFileIntoNewWindow: filePath: string -> line: int -> column: int -> bool

```

Comment:
Did you consider using an option here vs. using -1 as the "default" value?

Reply:
I did! I was ambivalent because using `Option` is awkward from C# and using `Nullable<T>` is awkward from F#. And of course `-1` is a "magic number", but I think of it as similar to `String.IndexOf` returning -1, which is a time-honored tradition in many programming languages.

In effect, there are three functionally-equivalent ways of doing the same thing and I chose the language-neutral way. I'm happy to switch to another method but I couldn't deduce which one would be preferred by myself.

Reply:
Yeah I've been frustrated with finding a good way to have default values that work equally well in C# and F#. I experimented a bit with F# optional parameters but they fall a bit short as they don't appear optional in C#. To use them I ended having to define two overloads:

1. One with all the parameters marked as optional
1. One with all all the optional parameters omitted that just forwards to the other 

The (2) version is callable from C#. Still though it's less than ideal 

https://github.com/jaredpar/VsVim/blob/master/Src/VimCore/CoreInterfaces.fs?utf8=%E2%9C%93#L870

I've been working on a proposal to get F# to emit their optional parameters in a way that would make it much easier to use from C#. But that's a ways away.

Reply:
I converted the two individual parameters to use `FSharpOption` which seems no worse than the `-1` approach!